collections/_article/january-jams-2024.md

```diff
@@ -0,0 +1,101 @@
+
+### 2. Finally Finish Something (FFS) 2024
+
+🔗 [https://itch.io/jam/finally-finish-something-2024](https://itch.io/jam/finally-finish-something-2024) 
+
+📅 January 1st  - February 4th
+
+Be honest, how high is your stack of unfinished games that you once were sure would see the light of day? This particular game jam caters to your new year’s resolution to”Finally Finish Something” and hence allows you to submit any game you have previously been working on, as long as you have used the month of January to complete the project.
+
+### 3. Queer Vampire Jam
+
+🔗 [https://itch.io/jam/queer-vampire](https://itch.io/jam/queer-vampire) 
+
+📅 January 10th  - February 20th
+
+While there are a plethora of “niche” game jams at any given time, it seems important to highlight at least one focused on the LGBTQIA+ community. Also, vampires are cool, and this interactive-fiction jam is not only still ongoing, but also kind of popular. Plenty of time to still submit your own bloodsucker romance!
+
+### 4. Game Dev Knock Out (GDKO) - Round 1
+
+🔗 [https://itch.io/jam/gdko-2024](https://itch.io/jam/gdko-2024) 
+
+📅 January 1st  - January 15th
+
+The Squid Games of the game dev world. The previous sentence is taken straight from their own branding, albeit it does seem nobody dies in this rendition and the price is a T-shirt. Every participant is sorted into one of five teams, spearheaded by the organizers themselves, which each adds a unique modifier to the common challenge.
+
+### 5. New Year, New Skills Game Jam
+
+🔗 [https://itch.io/jam/new-year-new-skills-game-jam](https://itch.io/jam/new-year-new-skills-game-jam) 
+
+📅 January 7th  - January 14th
+
+An explicitly beginner-friendly jam organized by [Bitter Sails Games](https://twitter.com/bittersails) that quickly rose to the top of itch.io’s game jam suggestions. Introductory presentations and mentoring by game industry veterans allowed for a guided experience, while the resource package prepared by the team is so comprehensive we suggest downloading it even after the fact.
+
+### 6. Godot Wild Jam #65
+
+🔗 [https://itch.io/jam/godot-wild-jam-65](https://itch.io/jam/godot-wild-jam-65) 
+
+📅 January 12th - January 21st
+
+A staple in our community that takes place during the second week of every month. In addition to the announced theme, three wildcards are released to provide an optional additional challenge. Most importantly, you are required to use the Godot Engine for this one ;)
+
+### 7. Pirate Software - Game Jam 14
+
+🔗 [https://itch.io/jam/pirate](https://itch.io/jam/pirate)
+
+📅 January 12th - January 27th
+
+A jam organized by Game Developer & Twitch Streamer [Pirate Software](https://twitter.com/PirateSoftware). Unique compared to the other events in this list is the requirement to provide a Game Design Document for your entry. On top of that, you can almost always find co-work companions in his live streams.
```

Comment:
```suggestion
A jam organized by game developer & Twitch streamer [Pirate Software](https://twitter.com/PirateSoftware). Unique compared to the other events in this list is the requirement to provide a game design document for your entry. On top of that, you can almost always find co-work companions in his live streams.
```
This is a matter of taste, but I think it looks better without so many capital letters.

Reply:
Agreed - sometimes my German-ness comes through an makes me want to capitalize everything xD